pyproject.toml

```diff
@@ -29,7 +29,7 @@ requires-python = ">=3.8"
 plotting = [
     # make sure a python 3.9 compatible scipy and matplotlib are selected
     "scipy>=1.5.4",
-    "matplotlib>=3.2.2,<3.6",
+    "matplotlib>=3.2.2",
```

Comment:
Should it be `>=3.6` now? Since we've invoked breaking changes.

Reply:
I ran the tests on matplotlib version 3.5.3 and they seem to work.